.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+services:
+  - mongodb
+node_js:
+  - node
+before_install:
+  - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+  - echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list
+  - sudo apt-get update
+  - sudo apt-get install -y mongodb-org=2.6.12 mongodb-org-server=2.6.12 mongodb-org-shell=2.6.12 mongodb-org-mongos=2.6.12 mongodb-org-tools=2.6.12
```

Comment:
Here's a bit more simple way of installing Mongo to Travis:

https://github.com/Trustroots/trustroots/blob/master/.travis.yml